test/fixtures/users.js

```diff
@@ -0,0 +1,6 @@
+const user1Token =
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJ0ZXN0QHRlc3QuY29tIiwiaXNBZG1pbiI6ZmFsc2UsImlhdCI6MTY2NDQ3NjQ2MH0._gAoZ1HLBTpwo0QBibvH11A4jFJDvE38I9_tGDyqNpc';
```

Comment:
Could it be created using a tool? https://stackoverflow.com/questions/64471965/mocking-jsonwebtoken-module-with-jest 
Is not necessary to implement it, just look for an alternative